samples/nrf9160/modem_shell/overlay-debug.conf

```diff
@@ -19,9 +19,6 @@ CONFIG_LOG_MAX_LEVEL=4
 # Increase log buffer size to prevent logs being dropped.
 CONFIG_LOG_BUFFER_SIZE=2048
 
-# LTE link control library
-CONFIG_LTE_LINK_CONTROL_LOG_LEVEL_DBG=y
```

Comment:
My motivation for removing LTE LC DBG logs from MOSH debug overlay is because there are too much of event dispatching logging:
```
<dbg> lte_lc_helpers: event_handler_list_dispatch: Dispatching events: 
<dbg> lte_lc_helpers: event_handler_list_dispatch:  - handler=0x02ECD9
```
Because of this, I always disable these manual when compiling so I cannot use debug overlay as is.

I don't know if this change is good or whether we should consider cleaning LTE LC logging, i.e., is it really needed.

Reply:
Could we have an exception this time and leave this in comments?

Reply:
That's fine for me. I'll add it.